src_kolko_krzyżyk/TicTacToe_expand.py

```diff
@@ -185,7 +185,35 @@ def main(self):
             else:
                 player = 0
 
-game = TicTacToe(5)
+def menu():
+    print('''\n\n\t"Simple game for two person in Tic Tac Toe.
+     Player decides how big the board is. Valid board (2 to n)
+     where board are matrix n x n. Good Luck and Have Fun."\n\n''')
+    # default matrix_choice
+    matrix_choice = 3
+    while True:
+        matrix_choice = input('Please enter size of martix from two to n: ')
+        try:
+            matrix_choice = int(matrix_choice)
+            break
+        except:
+            print("Dosen't entered number please try again" )
+
+    while True:
+        deletion_choice = input('Please enter number of deletions:'
```

Comment:
You can replace the phrase `number of deletions` with `winning condition number`. This name is more meaningful for me but you can try to convince me, that phrase is good enough.  You can also rename `deletion_choice` with `wining_condition`.